src/components/accordion/Accordion.spec.jsx

```diff
@@ -124,7 +125,7 @@ describe('<Accordion>', () => {
 
     expect(
       accordion
-        .find('Box')
+        .find(Box)
```

Comment:
this is something worth to remember, when forwardRef is used on element we can't find by string in tests :)